src/libs/ReportUtils.ts

```diff
@@ -3849,7 +3849,7 @@ function getWhisperDisplayNames(participantAccountIDs?: number[]): string | unde
  * Show subscript on workspace chats / threads and expense requests
  */
 function shouldReportShowSubscript(report: OnyxEntry<Report>): boolean {
-    if (isArchivedRoom(report)) {
+    if (isArchivedRoom(report) && !isWorkspaceThread(report)) {
```

Comment:
Could explain why we need to add check here?

Reply:
@tienifr Can you answer this question? Since I am about to fix `isWorkspaceThread` function.